PrivacyCookie/DemoBundleBanner.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\DemoBundle\PrivacyCookie;
+
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\URLAliasService;
+use EzSystems\PrivacyCookieBundle\Banner\Banner;
+use EzSystems\DemoBundle\PrivacyCookie;
+use Symfony\Component\Translation\Translator;
+
+/**
+ * Builds a Privacy Cookie Banner object based on a configured Content Object.
+ */
+class DemoBundleBanner implements DemoBundleBannerFactory
+{
+    public static function build( array $bannerOptions, LocationService $locationService, ContentService $contentService, URLAliasService $urlAliasService, Translator $translator )
+    {
+        $contentId = null;
+
+        // check if `Content` object path exists and if so, get associated `contentId`
```

Comment:
We usually favour `contentId` over `pathString`. It would also avoid the alias lookup below.